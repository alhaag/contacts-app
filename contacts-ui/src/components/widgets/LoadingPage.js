//third party
import React from 'react'
import CircularProgress from 'react-md/lib/Progress/CircularProgress'
// components
import PageWrapper from '../widgets/PageWrapper'


const LoadingPage = (props) => {
    return (
        <PageWrapper>
            <div style={styles.wrapper}>
                <CircularProgress id="circular-progress" />
            </div>
        </PageWrapper>
    )
}

const styles = {
    wrapper: {
        textAlign: 'center',
        padding: '60px 0',
    }
}

export default LoadingPage
