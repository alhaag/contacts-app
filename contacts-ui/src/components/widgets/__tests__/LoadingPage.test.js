import React from 'react'
import renderer from 'react-test-renderer'
import LoadingPage from '../LoadingPage';

test('Presentation LoadingPage component', () => {

  const component = renderer.create(
    <LoadingPage />,
  );
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()

})