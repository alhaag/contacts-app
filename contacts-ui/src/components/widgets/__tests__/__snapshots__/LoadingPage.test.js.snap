// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Presentation LoadingPage component 1`] = `
<div
  style={
    Object {
      "padding": "20px",
    }
  }
>
  <div
    style={
      Object {
        "padding": "60px 0",
        "textAlign": "center",
      }
    }
  >
    <svg
      aria-valuemax={100}
      aria-valuemin={0}
      className="md-progress md-progress--circular md-block-centered md-progress--circular-indeterminate"
      height={24}
      id="circular-progress"
      role="progressbar"
      style={Object {}}
      viewBox="0 0 66 66"
      width={24}
    >
      <circle
        className="md-circular-progress-path md-circular-progress-path--animated"
        cx="33"
        cy="33"
        r="30"
        strokeLinecap="round"
        strokeWidth="6"
        style={undefined}
      />
    </svg>
  </div>
</div>
`;
